.github/workflows/fuzzing.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Run fuzzer
         run: |
           cd out/results
-          ../norg_fuzzer -dict=../norg.dict -max_total_time=7200 -timeout=1 -rss_limit_mb=256 ./corpus
+          ../norg_fuzzer -dict=../norg.dict -max_total_time=7200 -timeout=1 ./corpus
```

Comment:
Why removing that memory limit?

Reply:
Hey, I discussed why I'm doing that on discord. It's a temporary thing just for now :). We'll readd it once I make sure that the parser itself doesn't hang in other places first. The issues that I'm having with the memory profiling is that no matter what tool I use I can never reproduce the OOM bug. Running Neovim with the file open and a memory profiler attached? Nothing. Profiling the `tree-sitter` cmdline tool? Also nothing. I've tried at least 4 different RSS profilers and none of them show me the large memory footprint that the fuzzer claims to have reached. Heck I've even tried running profilers on different operating systems through VMs and nothing either. I'm just stumped and so I've decided to let it go for a while if that's alright.

Reply:
Sure it is fine :) I was just wondering about it :) it seems that actions are finishing now.